Application/EdFi.Ods.Api/Security/Container/Modules/SecurityPersistenceModule.cs

```diff
@@ -27,40 +27,41 @@ public class SecurityPersistenceModule : Module
         private readonly IDictionary<Type, Type> _genericServiceByAuthorizationDecorator = new Dictionary<Type, Type>
         {
             // NHibernate authorization decorators
-            {typeof(IGetEntityByKey<>), typeof(GetEntityByKeyAuthorizationDecorator<>)},
-            {typeof(IGetEntitiesBySpecification<>), typeof(GetEntitiesBySpecificationAuthorizationDecorator<>)},
-            {typeof(IGetEntityById<>), typeof(GetEntityByIdAuthorizationDecorator<>)},
-            {typeof(IGetEntitiesByIds<>), typeof(GetEntitiesByIdsAuthorizationDecorator<>)},
-            {typeof(ICreateEntity<>), typeof(CreateEntityAuthorizationDecorator<>)},
-            {typeof(IDeleteEntityById<>), typeof(DeleteEntityByIdAuthorizationDecorator<>)},
-            {typeof(IUpdateEntity<>), typeof(UpdateEntityAuthorizationDecorator<>)},
-            {typeof(IUpsertEntity<>), typeof(UpsertEntityAuthorizationDecorator<>)},
+            {typeof(GetEntityByKeyAuthorizationDecorator<>), typeof(IGetEntityByKey<>)},
```

Comment:
Key/values were inverted in the dictionary here because the key should be the implementation, not the common interface (wasn't accounting for the possibility of multiple decorators implementing the same interface).